### ISSUE TITLE
[Spotlight] All Charting Texas Subpages

### ISSUE THREAD

**Issue by devangm** [TAMULib/exhibits#107]
**URL:** https://spotlight.library.tamu.edu/spotlight/charting-texas/about/exhibit-introduction
**WCAG 2.1 AA compliant? :** No
**Violation:** 1.4.3 Contrast Minimum (AA)
**Issues:** Contrast Error
**Notes:** Local nav bar items don't have enough contrast when not selected

**Comment by qtamu** [comment]
https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF80&bcolor=000000
